README.md

```diff
@@ -232,10 +232,10 @@ Thanks goes to these wonderful people
     <td align="center"><a href="https://github.com/chinmay17"><img src="https://avatars2.githubusercontent.com/u/7131231?v=4" width="100px;" alt="Chinmay Chaudhary"/><br /><sub><b>Chinmay Chaudhary</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=chinmay17" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/Sakthivel"><img src="https://avatars3.githubusercontent.com/u/205201?v=4" width="100px;" alt="Sakthivel Sengodan Sapient"/><br /><sub><b>Sakthivel Sengodan Sapient</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=sakthivel" title="Code">💻</a></td>
     <td align="center"><a href="https://github.com/vanya829"><img src="https://avatars0.githubusercontent.com/u/1397979?v=4" width="100px;" alt="vanya829"/><br /><sub><b>vanya829</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=vanya829" title="Code">💻</a></td>
-    <td align="center"><a href="https://github.com/lakhansamani"><img src="https://avatars3.githubusercontent.com/u/6964334?s=460&v=4" width="100px;" alt="lakhansamani"/><br /><sub><b>Lakhan Samani</b></sub></a><br /><a href="https://github.com/codesandbox/codesandbox-client/commits?author=lakhansamani" title="Code">💻</a></td>
```

Comment:
CLI removed [Lakhan Samani](https://github.com/lakhansamani) because he is not in .all-contributorsrc file. Both files should be resynced after adding him in .all-contributorsrc. Note sure what should I do from my end.

Reply:
@lakhansamani